src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -249,6 +251,7 @@ const formConfig = {
           uiSchema: ptsdWalkthroughChoice781.uiSchema,
           schema: ptsdWalkthroughChoice781.schema,
         },
+        ...formConfig781(3),
```

Comment:
Can you help me understand why this is hard coded to three?

Reply:
@rianfowler : Change this to PTSD_INCIDENT_ITERATION in constants file. thank you!